pom.xml

```diff
@@ -573,6 +573,7 @@
                     <groupId>org.asciidoctor</groupId>
                     <artifactId>asciidoctorj</artifactId>
                     <version>1.5.2</version>
+                    <scope>provided</scope>
```

Comment:
Yes, now that it ignores the documentation dependencies by default, I'll remove these build phase declarations.